### ISSUE TITLE
[Warning] Overriding environment GymV26Environment-v0 already in registry.

### ISSUE THREAD

**Issue by lazybuttrying** [Farama-Foundation/Gymnasium#308]
### Question

Always after call make(), those message came out. What can I do to hide it? I don't want to just hiding user warning, I hope to know how gymnasium works about registry .

```/usr/local/lib/python3.10/site-packages/gymnasium/envs/registration.py:521: UserWarning: WARN: Overriding environment GymV26Environment-v0 already in registry.
  logger.warn(f"Overriding environment {new_spec.id} already in registry.")
/usr/local/lib/python3.10/site-packages/gymnasium/envs/registration.py:521: UserWarning: WARN: Overriding environment GymV22Environment-v0 already in registry.
  logger.warn(f"Overriding environment {new_spec.id} already in registry.")
2023-02-03 11:42:06.617 Python[7314:60380] ApplePersistenceIgnoreState: Existing state will not be touched. New state will be written to /var/folders/ns/mrf2cq2x721d1khdmwz9dcl80000gn/T/org.python.python.savedState
2023-02-03 11:42:22.674 Python[7314:60380] TSM AdjustCapsLockLEDForKeyTransitionHandling - _ISSetPhysicalKeyboardCapsLockLED Inhibit```

**Comment by pseudo-rnd-thoughts** [comment]
Hi, we are working on fixing it but we require a couple of fixes in Gymnasium. We are hoping for this to be all done in the next week

**Comment by pseudo-rnd-thoughts** [comment]
This is fixed in shimmy `0.2.1`, `pip install --upgrade shimmy`
